openapi/atlas-api-transformed.yaml

```diff
@@ -13644,13 +13652,13 @@ paths:
             minLength: 1
             pattern: ^([a-zA-Z0-9][a-zA-Z0-9-]*)?[a-zA-Z0-9]+$
       responses:
-        "204":
+        "202":
```

Comment:
changing reponse code and body can be a semantinc break change for clients checking status codes

Reply:
remember this will usually be corrections, the api was returning the correct 202 but it was documented as 204

Reply:
i don't know in this case, but I've found some cases where status code really changed

Reply:
this change would trigger a breaking change upstream and we audit those changes, I can also say this particualr change was done by @colm-quinn :)